### DISCUSSION TITLE
Is it possible to remove a field value when condition becomes not met?

### DISCUSSION THREAD

**Discussion by YannickMol** [directus/directus#12021] [Archive]
Let's say I have a category dropdown field "fruits" (for example apples) and a sub category that has the condition set to only show choices based on the selected category (for example Jonagold, Elstar, Pink Lady etc.). Whenever I have selected a sub category, for example Jonagold, and I would select a different category (i.e. Bananas), the value of the sub category is not reset. Is there a way to achieve this?

**Comment by rijkvanzanten** [comment]
No, there's currently no way to have the field reset once the choice is no longer available. That being said, I think that would be a nice improvement to the dropdown interface itself. Whenever it's options change, and the current value is no longer one of the available options, the dropdown interface could set it's value to `null`. That would in turn also account for this problem 🙂
